Index: src/main/resources/application.yaml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>spring:\r\n  application:\r\n    name: discodeit\r\n  jpa:\r\n    open-in-view: false\r\n\r\ndiscodeit:\r\n  storage:\r\n\r\n    data-dir: data\r\n    # 이미지 파일\r\n    image-dir: C:/Users/ych61/IdeaProjects/codeit_sprint/2-sprint-mission/images\r\n# 로깅 설정\r\nlogging:\r\n  level:\r\n    root: INFO\r\n    com.sprint.mission.discodeit: DEBUG\r\n    org.springframework.web: INFO\r\n\r\njwt:\r\n  secret: your-very-strong-and-long-secret-key-please-change-this #
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/resources/application.yaml b/src/main/resources/application.yaml
--- a/src/main/resources/application.yaml	(revision 38847cc4648e6bf289386e163e1855ac9353ccf7)
+++ b/src/main/resources/application.yaml	(date 1744099905488)
@@ -4,6 +4,8 @@
   jpa:
     open-in-view: false
 
+
+
 discodeit:
   storage:
 
@@ -18,4 +20,4 @@
     org.springframework.web: INFO
 
 jwt:
-  secret: your-very-strong-and-long-secret-key-please-change-this #
\ No newline at end of file
+  secret: your-very-strong-and-long-secret-key-please-change-this
\ No newline at end of file
